src/OrbitGl/include/OrbitGl/SelectionData.h

```diff
@@ -0,0 +1,58 @@
+
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_SELECTION_DATA_H_
+#define CLIENT_DATA_SELECTION_DATA_H_
+
+#include "ClientData/CaptureData.h"
+#include "ClientData/ModuleManager.h"
+#include "ClientData/PostProcessedSamplingData.h"
+#include "OrbitGl/CallTreeView.h"
+
+// This is meant to hold the data needed to update the data tabs to a specific selection.
+class SelectionData {
```

Comment:
Since this is also holding the views, the name seems a bit misleading. But I also don't have a better idea.

Reply:
Yeah I'm definitely open to other ideas; but I would argue that the "view" name is the one that is more misleading. There aren't any ui elements in here as you can see all of the elements are returned const so nothing changes in them; it's just the data one needs to create the ui view.